src/engraving/dom/edit.cpp

```diff
@@ -1327,43 +1327,42 @@ bool Score::rewriteMeasures(Measure* fm, const Fraction& ns, staff_idx_t staffId
 //    Add or change time signature at measure in response
 //    to gui command (drop timesig on measure or timesig)
 //---------------------------------------------------------
-
-void Score::cmdAddTimeSig(Measure* fm, staff_idx_t staffIdx, TimeSig* ts, bool local)
+void Score::cmdAddTimeSig(Measure* targetMeasure, staff_idx_t staffIdx, TimeSig* targetTimeSignature, bool local)
```

Comment:
I'm not sure if `targetTimeSignature` is the best name, because `target` might be confused with "drop target" which would mean the time signature that is already in place and that the new time signature is being dropped upon.

How about `newTimeSig` instead? (same for related variables)

Reply:
newTimeSig makes sense! 

For measures and segment, targetMeasure and targetSegment still make sense, right? In this case the 'drop target' for the measure/segment is the same, so no confusion?